src/dpp/cluster/entitlement.cpp

```diff
@@ -62,15 +62,15 @@ void cluster::entitlements_get(snowflake user_id, const std::vector<snowflake>&
 
 	j["exclude_ended"] = exclude_ended;
 
-	rest_request_list<entitlement>(this, API_PATH "/applications", me.id.str(), "entitlements", m_get, j, callback);
+    rest_request_list<entitlement>(this, API_PATH "/applications", me.id.str(), "entitlements", m_get, std::string(j), callback);
```

Comment:
Again what's the point™

I'll stop commenting that because there are tens of these